src/components/encrypt-video.js

```diff
@@ -9,9 +9,14 @@ module.exports = React.createClass({
   },
   getInitialState() {
     return {
-      seventyFivePercentDone: false
+      nextPercentDone: 0.25
     };
   },
+  videoGaEvent(props) {
+    props.category = props.category || "Video";
+    props.label = props.label || this.context.intl.formatMessage({id: this.props.video.title});
```

Comment:
I thought about this but couldn't find a terribly pressing reason to do anything here.

Generally we havn't been doing tests, campaigns, or looking at data across multiple locales. We tend to look at data based on a campaign, and a campaign tends to target a locale. So a de campaign should not contain metrics from en-US, and we filter this via the url. So as long as the title is consistent for all titles in that url during the campaign, we should be fine.

As to what actually happens, if the locale doesn't exist, the user is directed to the closest match.

If the locale exists, but isn't complete (title isn't translated yet) it stores the id for the strings instead, which, is probably fine too.